internal/featuresupport/feature_support_test.go

```diff
@@ -504,6 +505,17 @@ var _ = Describe("V2ListFeatureSupportLevels API", func() {
 				Expect((*isNutanixIncompatibleWithUmn).getId()).To(Equal(umnFeature.getId()))
 			})
 
+			It("Features with restrictions - OCI and CMN", func() {
```

Comment:
Should we add a similar test for OCI + full iso ?

Reply:
I'll add one also for OCI + Full ISO

Reply:
Done!